### PR TITLE
On SUSE environments 'auditd' is part of package 'audit'

### PR DESCRIPTION
**Pull request by mcgege**

### CODE REVIEW

File: controls/package_spec.rb [hunk 1]
```diff
@@ -80,7 +80,7 @@
   impact 1.0
   title 'Install auditd'
   desc 'auditd provides extended logging capacities on recent distribution'
-  audit_pkg = os.redhat? ? 'audit' : 'auditd'
+  audit_pkg = (os.redhat? or os.suse?) ? 'audit' : 'auditd'
```
Comment (chris-rock) [inline controls/package_spec.rb]: Let's use
```
(os.redhat? || os.suse?)
```

**Comment by mcgege** [comment]
Uhh, tried to make rubocop happy, but it is very picky ... any ideas?

**Comment by chris-rock** [comment]
@mcgege You can run `rake lint` locally. Or just click on the travis link: https://travis-ci.org/dev-sec/linux-baseline/builds/247031354

**Comment by mcgege** [comment]
Google is your friend on error messages ...

**Comment by artem-sidorenko** [comment]
@mcgege thank you!

**Pull request merged**